src/app/views/query-runner/request/permissions/PanelList.tsx

```diff
@@ -49,7 +49,9 @@ const PanelList = ({ messages,
   const { permissionPanelStyles } = profileStyles(theme);
 
   useEffect(() => {
-    setPermissions(sortPermissions(fullPermissions));
+    if(searchValue === '' && groups.length === 0){
```

Comment:
This could also be null checked like so
```suggestion
    if(!searchValue && groups.length === 0){
```